superduperdb/db/base/metadata.py

```diff
@@ -19,92 +19,71 @@ def __init__(
         self.name = name
         self.conn = conn
 
-    @abstractmethod
-    def create_component(self, info: t.Dict):
-        pass
-
-    @abstractmethod
-    def create_job(self, info: t.Dict):
-        pass
-
-    @abstractmethod
-    def create_parent_child(self, parent: str, child: str):
-        pass
-
     @abstractmethod
     def drop(self, force: bool = False):
         """
         Drop the metadata store.
         """
         pass
 
-    @abstractmethod
-    def get_job(self, job_id: str):
-        pass
+    # ------------------ COMPONENTS ------------------
```

Comment:
Tried to clean up the structure of the existing meta-data stores, as well as the new one.